airbyte-ci/connectors/metadata_service/lib/metadata_service/templates/base.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
everything in the `templates` directory is just copied wholesale from `metadata_service/orchestrator/templates`.